pkg/util/proto/document.go

```diff
@@ -58,23 +59,21 @@ func VendorExtensionToMap(e []*openapi_v2.NamedAny) map[string]interface{} {
 // Definitions is an implementation of `Resources`. It looks for
 // resources in an openapi Schema.
 type Definitions struct {
-	models    map[string]Schema
-	resources map[string]string
+	resources map[string]Schema
```

Comment:
so my understanding is you removed "resources" and then renamed "models" to "resources". Why the later? I mean why not keep the name as models?